src/mongo/integration/standalone/dbclient_test.cpp

```diff
@@ -925,6 +953,57 @@ namespace {
         ASSERT_TRUE(doc["test"].boolean());
     }
 
+    TEST_F(DBClientTest, CopyDatabaseMONGODBCR) {
+        // only run this test if we have auth
+        if (serverGTE(&c, 2, 4) && !serverGTE(&c, 2, 7)) {
```

Comment:
The logic is that if the server is a new server, then we'll run the SCRAM-SHA-1 test instead of the MONGODB-CR test.